app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -0,0 +1,50 @@
+module Search
+  # TODO[@atsmith813]: Rename this to CommentSerializer once Elasticsearch is removed
+  class PostgresCommentSerializer < ApplicationSerializer
+    attribute :id, &:search_id
+
+    attributes :path, :public_reactions_count
+
+    attribute :body_text, &:body_markdown
+
+    attribute :class_name do |comment|
+      comment.class.name
+    end
+
+    attribute :highlight do |comment|
+      {
+        body_text: [comment.pg_search_highlight]
+      }
+    rescue PgSearch::PgSearchHighlightNotSelected
+      # This is needed because in Search::Postgres::Comment we only call the
+      # search if a term is provided. This means if a user searches with a
+      # blank term, we skip the line that executes .with_pg_search_highlight.
+      # Skipping this AND trying to call .pg_search_highlight raises an error
+      # which we ignore here - basically filling in highlights if they're
+      # there.
+    end
```

Comment:
Highlights in PostgreSQL FTS are handled differently - we have to access it _after_ the search whereas with Elasticsearch we specify it _before_ in the query itself. As a result, we need to add highlights in the serializer. Once we remove Elasticsearch we can rename this back to the "generic" `CommentSerializer`.